test/cli/more-projects_test.py

```diff
@@ -896,4 +896,23 @@ def test_project_file_nested(tmp_path):
         'cppcheck: error: nested Cppcheck GUI projects are not supported.'
     ]
 
-    assert_cppcheck(args, ec_exp=1, err_exp=[], out_exp=out_lines)
\ No newline at end of file
+    assert_cppcheck(args, ec_exp=1, err_exp=[], out_exp=out_lines)
+
```

Comment:
the official python style guide PEP recommends 2 blank lines between top level functions.